index.html

```diff
@@ -6,14 +6,41 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>Document</title>
+    <title>Jon F. Solo Build - Day 1 Lab</title>
     <link href="./styles.css" rel="stylesheet" type="text/css">
 </head>
 
 
 <body>
+   <div class="header"><h1>Jon's Solo Build Site</h1>
+    <h2>Some Info On My Cats</h2></div>
 
+    <p>I am following suggested advise and making my solo build website about
+    my cats.</p>
     
-    
+    <img src="./assets/butternpad.jpg" title="Butter and Padington napping">
+
+    <br><p>In order of Greatness, the cats I have owned as an adult are:</p>
+        <ol>
+            <li>Morris</li>
+            <li>Butter</li>
+            <li>Padington</li>
+        </ol>
+
+
+    <img src="./assets/butter.jpg">
```

Comment:
add alt="descirption of image goes here" to each image. This allows screen readers to read the image tag.